src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+
+import argparse
+import nidcpower
+import sys
+
+
+def example(
+    resource_name,
+    options,
+    lcr_frequency,
+    lcr_impedance_range,
+    cable_length,
+    lcr_voltage_rms,
+    lcr_dc_bias_source,
+    lcr_dc_bias_voltage_level,
+    lcr_measurement_time,
+    lcr_custom_measurement_time,
+    lcr_source_delay_mode,
+    source_delay,
+):
+    with nidcpower.Session(resource_name=resource_name, options=options) as session:
+        # Configure the session.
+        session.instrument_mode = nidcpower.InstrumentMode.LCR
+        session.lcr_stimulus_function = nidcpower.LCRStimulusFunction.VOLTAGE
+        session.lcr_frequency = lcr_frequency
+        session.lcr_impedance_range = lcr_impedance_range
+        session.cable_length = cable_length
+        session.lcr_voltage_amplitude = lcr_voltage_rms
+        session.lcr_dc_bias_source = lcr_dc_bias_source
+        session.lcr_dc_bias_voltage_level = lcr_dc_bias_voltage_level
+        session.lcr_measurement_time = lcr_measurement_time
+        session.lcr_custom_measurement_time = lcr_custom_measurement_time
+        session.lcr_source_delay_mode = lcr_source_delay_mode
+        session.source_delay = source_delay
+
+        with session.initiate():
+            session.wait_for_event(event_id=nidcpower.Event.SOURCE_COMPLETE, timeout=21.0)
```

Comment:
If there's a specific reason why 21 seconds is needed, a comment explaining why would be very useful. If no specific reason, I'd suggest using the default timeout value.

Reply:
Thanks @singerbistro for the explanation. I have added a comment and updated the timeout value to 5.0s in my latest commit:
```
# Low frequencies require longer settling times than the default timeout for
# wait_for_event(), hence 5.0s is set here as a reasonable timeout value
```